📖 clusterctl: initial set of documentation (#1994)
kubernetes-sigs/cluster-api: docs/book/src/clusterctl/overview.md

# Overview of clusterctl

The `clusterctl` CLI tool handles the lifecycle of a Cluster API management cluster.

## Day 1

The `clusterctl` user interface is specifically designed for providing a simple "day 1 experience" and a
quick start with Cluster API.

### Prerequisites

* Cluster API requires an existing Kubernetes cluster accessible via kubectl;

{{#tabs name:"tab-create-cluster" tabs:"Development,Production"}}
{{#tab Development}}

{{#tabs name:"tab-create-development-cluster" tabs:"kind,kind for docker provider,Minikube"}}
{{#tab kind}}

  ```bash
  kind create cluster --name=clusterapi
  kubectl cluster-info --context kind-clusterapi
  ```

See the [kind documentation](https://kind.sigs.k8s.io/) for more details.
{{#/tab }}
{{#tab kind for docker provider}}

> If planning to use the Docker infrastructure provider a custom kind cluster configuration is required in order to

Reviewer: ```suggestion
If you are planning to use the Docker infrastructure provider, a custom kind cluster configuration is required
```
